CHANGELOG.md

```diff
@@ -89,6 +152,9 @@ This release represents commits from 15 contributors! Thank you all.
 ### Deprecation Notices
 
 - Deprecate unmaintainable legacy table, `smart_drive_info` ([#7464](https://github.com/osquery/osquery/issues/7464), [#7542](https://github.com/osquery/osquery/pull/7542))
+- Drop shortcut_files table ([#7547](https://github.com/osquery/osquery/pull/7547))
+- Drop the ssdeep library and remove its hash support  ([#7525](https://github.com/osquery/osquery/pull/7525))
+- Remove libelfin and elf parsing tables ([#7524](https://github.com/osquery/osquery/pull/7524))
```

Comment:
These are already included in 5.2.3; even if 5.2.3 was in a separate branch, 5.3.0 should be still considered as a descendant, so it includes those.

Reply:
You're right! My grep missed those. Will remove!